libdebug/cffi/ptrace_cffi_build.py

```diff
@@ -73,6 +73,7 @@
     struct thread {
         int tid;
         struct user_regs_struct regs;
+        int signal;
```

Comment:
Could we rename this, just for clarity?
E.g. `signal_to_deliver`

Reply:
Yes, I will definitely do it. It is just a placeholder for now.